NOTICE

```diff
@@ -1,4 +1,4 @@
-bot-unfurl
-Copyright 2016 Fintech Open Source Foundation
+unfurl bot - FINOS
+Copyright 2016-2019 Symphony LLC
```

Comment:
This copyright notice is incorrect in (at least) two ways:

1. It only requires a year of first publication - [copyrights are inherently open ended into the future](https://en.wikipedia.org/wiki/Copyright_notice#Form_of_notice_for_visually_perceptible_copies)
2. The copyright holder of this code is FINOS, _not_ Symphony LLC, and as the original author I am opposed to any reassignment of copyright away from FINOS (if that's what's being proposed)